.circleci/config.yml

```diff
@@ -67,84 +37,74 @@ jobs:
     working_directory: ~/grakn
     steps:
       - checkout
-      - bazel_install
-      - bazel_add_rbe_credential
-      - bazel:
-          command: bazel test //common/... --test_output=errors
+      - install_bazel
+      - run: bazel run @graknlabs_build_tools//ci:install-bazel-rbe-credential
+      - run: bazel run @graknlabs_build_tools//ci:run-bazel bazel test //common/... --test_output=errors
 
   test-console:
     machine: true
     working_directory: ~/grakn
     steps:
       - checkout
-      - bazel_install
-      - bazel_add_rbe_credential
+      - install_bazel
+      - run: bazel run @graknlabs_build_tools//ci:install-bazel-rbe-credential
```

Comment:
can we hide this `install-bazel-rbe-credentials` inside `install_bazel`, ganesh?